lib/container/ModuleFederationPlugin.js

```diff
@@ -63,7 +70,15 @@ class ModuleFederationPlugin {
 		if (!hooks) {
 			hooks = {
 				addContainerEntryDependency: new SyncHook(["dependency"]),
-				addFederationRuntimeDependency: new SyncHook(["dependency"])
+				addFederationRuntimeDependency: new SyncHook(["dependency"]),
+				generateFederationStartup: new SyncBailHook(["params"]),
```

Comment:
This hook is never tapped, i think that my plugin is added too late after whatever uses this, so maybe this should be renamed and added to javascript modules plugin so that generateEntry in startup helpers can call it and its tapped ahead of time / registered in time for the tap?

Reply:
I think we already have a similar hook - `JavascriptModulesPlugin.hooks.renderStartup`?

Reply:
Render startup is too wide, i need something inside render startup. See other comments.